subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/swi.h

```diff
@@ -48,4 +48,45 @@ static inline void hal_swi_job_pend(void)
 	NVIC_SetPendingIRQ(HAL_SWI_JOB_IRQ);
 }
 
+#elif defined(CONFIG_SOC_SERIES_NRF53X)
+
+/* SW IRQs required for the nRF5 BLE Controller. */
+#if defined(CONFIG_BT_LL_SW_SPLIT)
+
+static inline void hal_swi_init(void)
+{
+	/* No platform-specific initialization required. */
+}
+
+/* Split architecture uses max. two SWI */
+#define HAL_SWI_RADIO_IRQ     NRF5_IRQ_EGU0_IRQn
+#define HAL_SWI_WORKER_IRQ    NRF5_IRQ_RTC0_IRQn
```

Comment:
Definitions of these two `NRF5_IRQ_*` labels were removed by [this commit](https://github.com/zephyrproject-rtos/zephyr/commit/35754375bc2447b4cd4c429d2662a2e96ed5b8cd).
Use `EGU0_IRQn` and `RTC0_IRQn` instead.

Reply:
Great, fixing this as well.